README.md

```diff
@@ -362,10 +362,10 @@ $ git reset --hard HEAD
 This will remove all local untracked files, so only files tracked by Git remain:
 
 ```sh
-$ git clean -fdx
+$ git clean -fd
 ```
 
-WARNING: -x will also remove all ignored files!
+`-x` will also remove all ignored files!
```

Comment:
Remove the `!`, and I think this is good to go!

Reply:
Removed 😃